INSTALL.txt

```diff
@@ -25,11 +25,9 @@ It also contains different para-virtualized (PV) backends running inside it in o
 (without HW assigned) with different services such as Audio, Display, Net, Block, etc.
 3. Android based domain named "DomA". It has different PV frontends running and doesn't have any HW assigned except GPU.
 The latter is shared between "DomD" and "DomA" in current setup. 
-
-It worth to mention that IPMMU for GPU is enabled in case of H3 ES3.0 only. For other supported SoCs the PVRKM
-performs IPA to PA translations using a hypercall to hypervisor.
-
-The ARM TF was patched to bring up all available CPU cores.
+- or -
+4. Linux based simple domain named "DomU". It has different PV frontends running and doesn't have any HW assigned except GPU.
+The latter is shared between "DomD" and "DomU" in current setup.
```

Comment:
Item 4 contradicts line 19 "There are three domains which ...". Need some alignment.

Reply:
ok, will rewrite in the following way:
3. Either Android based domain named "DomA" or Linux based domain named "DomU".
Both have different PV frontends running and don't have any HW assigned except GPU.
The latter is shared between "DomD" and "DomA" ("DomU") in current setup.